portal-network.md

```diff
@@ -55,8 +61,11 @@ TODO
 
 ### Transaction Sending: Cooperative Transaction Gossip
 
-TODO
+The goal of the transaction gossip network is to make sure all new transactions are made available to the miners so that they can be included in a block.
+
+Stateless clients should be able to declare how many transactions they want to process out of the set of all unmined and valid transactions (called _mempool_) based on the amount of resources they have, and should only receive that many transactions from other nodes.
```

Comment:
I would change the phrasing minorly here.  A literal reading of this might leave the interpretation that participants are configuring *exactly* how many transactions they will be receiving.  In reality, they are tuning what *percentage* of the total mempool they will receive.